doc/Actions.md

```diff
@@ -34,10 +34,22 @@ Remove characters in metadata title fields.
 
 - `pattern` (required): A regular expression to search for. Example: `<<(.*?)>>` would remove "<<" and ">>" from titles.
 
+## onDemandAwtPdfboxSingleFileConvertAction
```

Comment:
I did not add these actions as they need the required parameter `requestUrl`. So this action and the other onDemand* one might be in the fileserver only.
If you want to add it anyway, you should add the required parameters here.

Reply:
You're right, it doesn't really make sence to perfom them in another context since they only return an InputStream. I deleted them from the documentation.